### ISSUE TITLE
Chrome logs and other LevelDB-related cleaning

### ISSUE THREAD

**Issue by EvgenKo423** [MoscaDotTo/Winapp2#320]
Chrome `[Logs *]` cleaning entry is invalid and will probably cause a lot of damage (even if you won't notice it in some way)!

Folders like `User Data\Default\Extension State` is a `LevelDB` database and `.log` files in there are not a usual logs. They are buffers of an actual data stored in those databases and you shouldn't mess with them as it will most likely corrupt the database!
You should delete the whole contents of such folders if you wanna clean some database. Otherwise only `LOG` and `LOG.old` files (the actual logs) can be deleted. There will be no need in exclude entries if you only clean the actual logs.

See [here](https://github.com/google/leveldb/blob/master/doc/impl.md) for more info.

**Comment by APMichael** [comment]
Thanks for your clarification.

I don't use Chrome, therefore I can't really comment this. All I can say is that the entry is very old and that we never have got issue reports since we have added the Exclude Keys.

@MoscaDotTo What's your opinion? Should we merge the entries [Logs \*] and [Logs Old \*], change the pattern to |LOG;LOG.old|RECURSE and remove the ExcludeKeys? But I think not everybody will be happy with these changes. Or will you discuss it in the community forum first?

**Comment by EvgenKo423** [comment]
Same about `[Web Applications *]` entry: it deletes part of useful files.
BTW, it's not needed as it's a part of `[HTML5 Storage *]` entry.

**Comment by APMichael** [comment]
I opened a new pull request.

@MoscaDotTo I will wait for your opinion/decision before I will merge the pull request.

**Comment by MoscaDotTo** [comment]
I think we should avoid unexpectedly breaking configurations as best we can. Especially for things like Browser Extensions where for most users, tracking down why they lost their settings is probably an impossibility. 

Perhaps a more dangerous entry in winapp3 for people who supremely don't want the log files, or otherwise don't have any extensions whose configurations would be damaged by this entry?

**Comment by EvgenKo423** [comment]
Just to clarify: it's not only about extensions. It's used for various things across the user profile.

**Comment by APMichael** [comment]
Thanks for your reply. Good, then I will merge the pull request tomorrow.

Yes, we could add the dangerous version of the entry [Logs /*] to Winapp3.ini. But I think that the original entry was created without the necessary knowledge (that the .log files are not usual logs, but important database files). Thanks to EvgenKo423 we are now smarter.

**Comment by MoscaDotTo** [comment]
Agreed. After reviewing the LevelDB documentation this seems to be the case.

**Comment by APMichael** [comment]
The pull request has been merged. I hope that the Chrome entries are ok now.
